WayMovies/Model/Request.swift

```diff
@@ -0,0 +1,305 @@
+//
+//  Request.swift
+//  WayMovies
+//
+//  Created by Hersh Nagpal on 1/22/20.
+//  Copyright © 2020 Hersh Nagpal. All rights reserved.
+//
+
+import Foundation
+
+struct Request {
```

Comment:
this is fine, but the name of the class suggest this is a request, where the function of this struct is more like a request manager.